python/ray/data/tests/test_actor_pool_map_operator.py

```diff
@@ -211,6 +263,35 @@ def test_pick_all_max_tasks_in_flight(self):
         # Check that the next pick doesn't return an actor.
         assert pool.pick_actor() is None
 
+    def test_pick_ordering_restarting(self):
+        # Test that pick ordering is honored by restarting actors
```

Comment:
This test doesn't seem useful. `pick_actor` already excludes restarting actors. The ordering is just an internal implementation detail.

Reply:
This test ensures the pick_actor implementation honors _is_restarting flag and excludes it from picking choices. 

Thought it would good to have the coverage for the unit test.

Reply:
> pick_actor implementation honors _is_restarting flag and excludes it from picking choices.

but doesn't test_restarting_to_alive already cover this?

Reply:
Ah ok yes. This tests with more that one active actors, but yes, test_restarting_to_alive covers restarting actors and pick_actor case. I can remove this.